core/src/main/java/hudson/cli/CLICommand.java

```diff
@@ -353,34 +321,7 @@ protected CmdLineParser getCmdLineParser() {
      */
     @Deprecated
     public Channel checkChannel() throws AbortException {
-        if (channel==null)
-            throw new AbortException("This command is requesting the deprecated -remoting mode. See https://jenkins.io/redirect/cli-command-requires-channel");
-        return channel;
-    }
-
-    /**
-     * Loads the persisted authentication information from {@link ClientAuthenticationCache}
-     * if the current transport provides {@link Channel}.
-     * @deprecated Assumes Remoting, and vulnerable to JENKINS-12543.
-     */
-    @Deprecated
-    protected Authentication loadStoredAuthentication() throws InterruptedException {
```

Comment:
Probably we do not care, bu https://github.com/jenkinsci/jenkins.py/blob/cbfc2ec231b447638d2d3e3bf82e3bc7bb160549/python-wrapper/src/main/java/jenkins/python/expoint/CLICommandPW.java

Reply:
AFAICT `jenkins.py` simply mirrors everything in Jenkins; it is not an actual plugin _using_ the API.